prometheus/utils.go

```diff
@@ -26,6 +27,8 @@ func connectRaw(ctx context.Context, cc *connection.ConnectionCache, c *plugin.C
 
 	// Prefer config settings
 	prometheusConfig := GetConfig(c)
+
+	address := os.Getenv("PROMETHEUS_URL")
```

Comment:
```suggestion
	address := os.Getenv("PROMETHEUS_ADDRESS")
```

For consistency with the config arg name, I think this env name would be better (assuming there's no other standards that Prometheus or related tools use already).

It's a bit longer, but having the config arg name match its env var's name makes it more intuitive